### PR TITLE
Rename student

### PR DESCRIPTION
**Pull request by teamotitan**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -49,37 +49,37 @@ public class AddCommand extends Command {
...
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New student added: %1$s";
+    public static final String MESSAGE_DUPLICATE_STUDENT = "This student already exists in the address book";
 
-    private final Person toAdd;
+    private final Student toAdd;
 
     /**
-     * Creates an AddCommand to add the specified {@code Person}
+     * Creates an AddCommand to add the specified {@code Student}
```
Comment (TY1Fan) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:58]: Nice! Great to see that you have changed the header comments too!

**Review by TY1Fan — APPROVED** [review]
LGTM!

**Review by DennieDan — APPROVED** [review]
LGTM!

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM. Good work on changing all instances of person to student.

**Pull request merged**